pkg/process/config/data_scrubber_test.go

```diff
@@ -419,7 +419,7 @@ func benchmarkRegexMatching(nbProcesses int, b *testing.B) {
 	scrubber.AddCustomSensitiveWords(customSensitiveWords)
 
 	for i := 0; i < nbProcesses; i++ {
-		runningProcesses = append(runningProcesses, foolCmdline)
```

Comment:
I don't think this was initially intended. The benchmark was running against nil values most of the times.